message_definitions/v1.0/common.xml

```diff
@@ -873,6 +882,26 @@
         <param index="6">Longitude</param>
         <param index="7">Altitude (ground level)</param>
       </entry>
+      <entry value="86" name="MAV_CMD_NAV_DELAY_BUTTON">
+        <description>Wait for a button state change. This is intended to be used as a method to allow a takeoff delay using a custom button.</description>
+        <param index="1">button bitmask</param>
```

Comment:
Maybe parameter 6 should be used? This would actually allow you to set a full 32 buttons instead when encoded as a `_INT` type.